frontend/src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'https://vaccination.mild.blue/'
+  apiUrl: 'https://vaccination.mild.blue/api'
```

Comment:
Asi spíš to ukazovat na staging protože to je master, tahle produkce se nereleasuje tak často

Reply:
jaká je url na staging prosím?

Reply:
https://covid-vaxx.stg.mild.blue/ (pardon)